util/parser.go

```diff
@@ -3,8 +3,11 @@ package util
 import (
 	"fmt"
 	"strconv"
+	"time"
 )
 
+var RecordDateTimeFormat = "2006-01-02 15:04:05 MST"
```

Comment:
I _think_ this is similar to RFC3339, and there is a constant here: https://cs.opensource.google/go/go/+/refs/tags/go1.17.3:src/time/format.go;l=101

That matches what the current API is returning, they even have a `RubyDate` constant too if we wanted to use that.

Reply:
Currently returned by our ruby/rails(and this RecordDateTimeFormat) availability status listener: `2021-12-21 11:25:36 UTC`
time.RFC3339: `2021-12-21T11:26:40Z`
time.RubyDate: `Tue Dec 21 11:27:53 +0000 2021`

Of course I would rather use existing constant but it is not same - so it could potentially break systems which are using it and it is main reason why I defined own.

Let me know whether we can change it - but as I am looking on format constants - I still prefer `RecordDateTimeFormat` :) it has good readability